### PR TITLE
Feature/harmonisation documentation

### PR DESCRIPTION
**Pull request by samleeflang**
Added documentation on how the harmonisation and MIDS calculation works.
Harmonisation happens in the translator service.
MIDS calculation happens on the harmonised model in the Digital Specimen Processing Service.

### CODE REVIEW

**Comment by samleeflang** [comment]
Feel free to add additional reviewers.

File: harmonisation/intro.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Introduction to harmonisation
+
+For DiSSCo to provide be fully machine actionable units (FDOs) there needs to be a standardized datamodel.
```
Comment (TomDijkema) [inline harmonisation/intro.md:3]: 'to provide be fully machine actionable..' this reads weird

File: harmonisation/intro.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+This datamodel provides a way to know which information to expect and where to expect it.
+The data standard we will call the open Digital Specimen (openDS) and it build upon existing standards.
```
Comment (TomDijkema) [inline harmonisation/intro.md:5]: This sentence also is weird to read, do you mean: 'The data standard, which we will call the open Digital Sepcimen (openDS), builds upon the existing standards.'

File: harmonisation/intro.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+This openDS will also form the basis of the FDO in the form of the FDO Type.
+
+Data coming into DiSSCo will not conform to the openDS data specification.
+They come from a range of different data providers (often CMS's) in a range of different data standards.
+In first instance we have focussed on two often used ways of data exposure, Darwin Core Archives (DWCA) and biocase.
```
Comment (TomDijkema) [inline harmonisation/intro.md:10]: biocase -> BioCASe

File: harmonisation/intro.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+But even within a single data standard different elements can be filled in differently.
+Additionally we have the elements expected and defined by MIDS, specifically focussed on collection specimens.
+The elements in MIDS do not always exact match a term in the existing data standards.
+
+Within the translator we will harmonise the data into the openDS data specification.
+The terms of openDS have been determined based on the MIDS elements and checked for existing terms within DWC and ABCD(EFG).
+If no matching term was identified in DWC and ABCD(EFG) a new term under the opends (ods) ontologie was proposed.
+All terms are subject to change and are open for disucssion.
+
+In this folder we will show how we harmonised the information in DWCA and biocase endpoints in to a single data specification, openDS.
```
Comment (TomDijkema) [inline harmonisation/intro.md:21]: biocase -> BioCASe

**Review by TomDijkema — COMMENTED** [review]
For me, this documentation is very useful to get some sight on what is going on in the MIDS specification and OpenDS. I think this format and structure for explaining each term, within its own file, is a good practice.

File: harmonisation/specimen/biostratigraphicTerms.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# Biostratigraphic terms
+
+This is a set of terms which are combined in this single file.
+This is mainly te reduce the amount of files.
```
Comment (southeo) [inline harmonisation/specimen/biostratigraphicTerms.md:4]: This is **to** reduce the amount of files

Comment (southeo) [reply harmonisation/specimen/biostratigraphicTerms.md:4]: Reduce the files in the backend?

**Review by southeo — CHANGES_REQUESTED** [review]
Good documentation, very informative and useful!
One thing: there are a lot of missing commas/punctuation. I made some comments, then some more, and then it quickly got out of hand. When you push this, I'll make a PR and make the changes myself. It seems like Dutch is a lot more fast-and-loose with commas :laughing: This is also a topic a lot of native English speakers have trouble with!

Maybe in the future we can implement Language Tool :)

File: harmonisation/data-mapping.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
+# Mapping Json
+
+Before any automated mapping is being made the system will first see if the user (data administrator) provided any explicit information on the mapping.
```
[the remark this answers is not in the document]
Comment (samleeflang) [reply harmonisation/data-mapping.md:3]: Will change this to, **will be** as we can do it before we look at the mapping

File: harmonisation/data-mapping.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+This information can be in two forms:
+- Setting of default values 
+- Setting of an explicit field mapping
+
+The application will first check if a default value has been set, then if an explicit field mapping was set, then it will use the logic described in the specific terms. 
+
+## Setting of default values
```
[the remark this answers is not in the document]
Comment (samleeflang) [reply harmonisation/data-mapping.md:10]: Will change

File: harmonisation/data-mapping.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+A data administrator can set default values for any attribute in the harmonised data model.
+Whenever this mapping it used, all attributes for a particular field will be set.
+This means that for example all information in the dataset has the same license but this license is not part of the dataset, it can be set as a default value.
+The format is `{harmonised key} : {fixed value}`
+Before the translator does anything for a particular field it will first check if a default value has been set.
+
```
[the remark this answers is not in the document]
Comment (samleeflang) [reply harmonisation/data-mapping.md:16]: Will change

File: harmonisation/intro.md (hunk 1 above)
Comment (samleeflang) [reply harmonisation/intro.md:3]: Comma added

File: harmonisation/intro.md (hunk 2 above)
Comment (samleeflang) [reply harmonisation/intro.md:5]: Changed

File: harmonisation/specimen/biostratigraphicTerms.md (hunk 5 above)
Comment (samleeflang) [reply harmonisation/specimen/biostratigraphicTerms.md:4]: Will change

**Review by TomDijkema — APPROVED** [review]

**Pull request merged**